NashTeamAlpha/BangazonWeb
# Submit Order Button

## Pull Request Description

Submit order button routes users to order completed page. Upon pressing the button it patches the order to the database.

## Steps to Test

1. Check out orderCompleted branch.
1. Nuke your migrations and database file.
1. Export DB path.
1. Add migrations.
1. Update DB.
1. dotnet run
1. Comment any issues.

## Link to Feature Ticket

#155
